hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/read/TestECBlockReconstructedStripeInputStream.java

```diff
@@ -354,8 +355,8 @@ public void testErrorThrownIfBlockNotLongEnough() throws IOException {
       try {
         ecb.readStripe(bufs);
         Assert.fail("Read should have thrown an exception");
-      } catch (IOException e) {
-        Assert.assertTrue(e.getMessage().matches("^Expected\\sto\\sread.+"));
+      } catch (InsufficientLocationsException e) {
```

Comment:
Instead of empty catch, we could also add header annotation in test (expected = InsufficientLocationsException.class). But I am not worried though.

Reply:
Yea, I am aware there is some JUnit magic to do this sort of thing, but find it easier to understand like this and avoid the "magic" annotations